src/languages/ruby.coffee

```diff
@@ -29,7 +29,10 @@ module.exports = {
       type: 'string'
       default: null
       description: "Indentation character"
-      enum: [" ", "\t"]
+      enum: [
+        {value: " ", description: "Space"}
+        {value: "\t", description: "Tab"}
```

Comment:
👍 Looks like Atom supports this structure: https://atom.io/docs/api/v1.6.0/Config

![image](https://user-images.githubusercontent.com/1885333/36882317-783e7ea6-1da9-11e8-8a97-9aed68e0e737.png)